public/app/features/dashboard/components/SubMenu/DashboardLinksDashboard.tsx

```diff
@@ -18,7 +18,17 @@ export const DashboardLinksDashboard: React.FC<Props> = props => {
   const { link, linkInfo } = props;
   const listRef = useRef<HTMLUListElement>(null);
   const [opened, setOpened] = useState(0);
-  const resolvedLinks = useResolvedLinks(props, opened);
+  let resolvedLinks = useResolvedLinks(props, opened);
+
+  // TODO: remove it ASAP after migration transition period is finished
```

Comment:
Better drop a ticket, or it's easy to forget

Reply:
Added ticket PMM-7736, thanks